composer.json

```diff
@@ -23,7 +23,7 @@
   "require": {
     "behat/behat": "~3.2",
     "behat/mink": "~1.5",
-    "behat/mink-extension": "~2.0",
+    "friends-of-behat/mink-extension": "^2",
     "behat/mink-goutte-driver": "~1.0",
```

Comment:
I need also `behat/mink-goutte-driver:~2.0`. Could we update to `~1.0 || ~2.0`

Reply:
I *think* I tried that, but don't remember why I didn't use that, possibly some conflict? 

this was merged, so will require a new MR anyway. That said, I think the path forward is to use browserkit directly once https://github.com/FriendsOfBehat/MinkExtension/pull/14 lands. Per https://github.com/minkphp/MinkGoutteDriver/releases/tag/v2.0.0, that's just a thin wrapper and you're supposed to switch over.

Reply:
@Berdir  I've created #610 (didn't test locally)

Reply:
@Berdir this is blocking me adopting PHP 8.1

> (...) that's just a thin wrapper and you're supposed to switch over.

To switch where? As Drupal Extension have this dependency, I suppose I need to switch here. But I can't find where

Reply:
the browserkit driver, as described in the 2.0.0 release notes: https://github.com/minkphp/MinkGoutteDriver/releases/tag/v2.0.0. And https://github.com/FriendsOfBehat/MinkExtension/pull/14 was just merged and released two days ago, so that might be possible now.

I guess that will require changes in this project, but it seems to be the mid/longterm path forward.

Reply:
@Berdir, what you're saying is that we should replace `behat/mink-goutte-driver` with `behat/mink-browserkit-driver` because the former is only a wrapper around the latter. Well, I did this in https://github.com/jhedstrom/drupalextension/pull/610 but it seems goutte driver is still requested by `friends-of-behat/mink-extension` as you can see in the test failure. I don't see anything we can do here, in drupal-extension

Reply:
> but it seems goutte driver is still requested by friends-of-behat/mink-extension as you can see in the test failure.

friends-of-behat/mink-extension only has a require-dev dependency on goutte-driver.

Not sure what's the earlier and the current MR, but the test fails I see currently in the MR are because of conflicts with older PHP versions (symfony 6 requires PHP 8.1) and drupal 9 vs symfony versions conflicts. Which makes sense, my guess is that this approach won't work until a new major version that can require Drupal 10 or something.

The current state works for me on php 8.1 beside those initial guzzle deprecation messages. I'm not against requiring or maybe additional supporting v2 so that you can pick a matching version depending on drupal and php version, just that it's not a long-term solution.